platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpointTest.java

```diff
@@ -70,7 +70,7 @@
 
     private static final String USING_DEFAULT = "Filename not found, using default.";
 
-    private Logger logger = LoggerFactory.getLogger(ApplicationUploadEndpoint.class);
+    private Logger LOGGER = LoggerFactory.getLogger(ApplicationUploadEndpoint.class);
```

Comment:
We're starting to get out of scope here, but it doesn't look like this LOGGER is being used in the class